CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+### Changed
+
+## [0.1.1]
+
+### Added
+
+- Support installation of dbt versions >= 0.20.0 in a Python 3.9 environment.
+
+### Changed
```

Comment:
Having placeholder headers for "Unreleased" makes sense to me, but for these concrete releases let's only have headers if there are things to list under them.